neo/renderer/Vulkan/RenderBackend_VK.cpp

```diff
@@ -1508,7 +1503,7 @@ void idRenderBackend::Init()
 
 	// DG: make sure SDL has setup video so getting supported modes in R_SetNewMode() works
 	// SRS - Add OSX case
-#if ( defined(__linux__) || defined(__APPLE__) ) && defined(USE_VULKAN)
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
&& defined(USE_VULKAN) is still required because VKimp_* is only for Vulkan.

Reply:
&& defined(USE_VULKAN) seemed redundant in this case since RenderBackend_VK.cpp is only pulled in for Vulkan builds.  For OpenGL builds, RenderBackend_GL.cpp is used instead.  In any case, that part of the pull request is a minor issue and could be left as is.  The rest should be valid.  Do you want me to revoke and resubmit, or can you merge the rest of the pull request?